### DISCUSSION TITLE
mac 版app无法解锁

### DISCUSSION THREAD

**Discussion by BaiLuoYan** [1715173329/UnblockNeteaseMusic#427] [Q&A] [answered]
之前一直使用自己编译的代码跑的服务，但是上次 mac 版 app 升级版本之后无法解锁了，除了降级app版本之外，有其他办法解决吗

**Comment by pan93412** [comment]
Will investigate it later.

**Comment by pan93412** [comment]
<img width="1114" alt="截圖 2021-12-27 下午6 11 04" src="https://user-images.githubusercontent.com/28441561/147462058-de7bb102-0b5a-4539-9e4e-30b5fc109161.png">

<img width="780" alt="截圖 2021-12-27 下午6 17 19" src="https://user-images.githubusercontent.com/28441561/147462071-eaa5d300-fd11-479d-b243-899b4aa4299d.png">

<img width="653" alt="截圖 2021-12-27 下午6 17 40" src="https://user-images.githubusercontent.com/28441561/147462079-76fa5c0b-124d-4ccc-b168-8783654b09b1.png">

我這裡版本是 2.3.7 build 868。使用 Proxy 代理伺服器，運作挺正常的。方便提供更詳細的資訊嗎？謝謝！

**Comment by BaiLuoYan** [comment]
@pan93412 
![image](https://user-images.githubusercontent.com/8652307/147740776-760c6c11-d05a-4b5b-be56-0274b953de93.png)

![image](https://user-images.githubusercontent.com/8652307/147740722-ef155878-ced2-41b7-b5dc-00291d0e60f5.png)

![image](https://user-images.githubusercontent.com/8652307/147741140-46aa31f4-fbdb-4c51-8139-c6a0db13e5e1.png)

奇怪，我的代理都收不到请求了

**Comment by BaiLuoYan** [comment]
@pan93412 我本地装有 nginx，刚开始以为是 nginx 转发配置影响到代理服务，所以把 nginx 停了，但还是收不到请求。
我用 https://github.com/nondanee/UnblockNeteaseMusic 这项目试了一下，能收到请求，但全是这种日志
![image](https://user-images.githubusercontent.com/8652307/147742628-9d26248e-637e-4554-88aa-28d24d455194.png)

**Comment by pan93412** [comment]
在 node 指令前面加上 `LOG_LEVEL=debug`，並傳給我完整操作的日誌內容？

**Reply by BaiLuoYan** [reply]
@pan93412 
![image](https://user-images.githubusercontent.com/8652307/147752758-3621cf58-5c19-4231-ab70-5090c5695730.png)

**Reply by pan93412** [reply]
你用 Safari 進入 https://music.163.com 會顯示什麼？

**Reply by BaiLuoYan** [reply]
网易云音乐主页

**Reply by pan93412** [reply]
那憑證顯示什麼？

**Reply by BaiLuoYan** [reply]
什么凭证？

**Reply by pan93412** [reply]
大陸應該叫「證書」，也就是 HTTPS Certificate。假如 UNM 運作正常，正常來講應該長這樣：

<img width="1092" alt="截圖 2021-12-30 下午8 45 11" src="https://user-images.githubusercontent.com/28441561/147753105-a25d509a-44c6-435e-a269-50243b521353.png">

**Reply by BaiLuoYan** [reply]
![image](https://user-images.githubusercontent.com/8652307/147753187-eee9df6c-8c72-4375-ab27-f89cf791cc80.png)
是我的证书错误了吗？

**Reply by pan93412** [reply]
> ![image](https://user-images.githubusercontent.com/8652307/147753187-eee9df6c-8c72-4375-ab27-f89cf791cc80.png)
> 是我的证书错误了吗？

請遵循本步驟重新安裝憑證：https://github.com/UnblockNeteaseMusic/server/discussions/426

**Reply by BaiLuoYan** [reply]
谢谢，我重新安装一下:)

**Reply by BaiLuoYan** [reply]
好了，谢谢你，之前的证书失效了>_<

**Comment by pan93412** [comment, marked as answer]
請根據 #426 重新安裝憑證。
